pcgc/opsmanager_client.go

```diff
@@ -0,0 +1,77 @@
+package pcgc
+
+import (
+	"bytes"
+	"encoding/json"
+)
+
+type opsManagerAPI struct {
+	BasicHTTPOperation
+
+	resolver URLResolver
+}
+
+// OpsManagerUser request object which identifies a user
+type OpsManagerUser struct {
```

Comment:
Any reason for using `OpsManagerUser` an not just `User` from what I've seen so far go prefers short names to declarative ones

Reply:
You'd use `User` if the package was named `opsmanager`, as due to go syntax, when using the struct you'd be forced to write `opsmanager.User`;  in this case, we'd use `pcgc.OpsManagerUser` making it a bit more explicit.

That said, it might make sense to create multiple packages and move these types around.
I think it's too early to tell (not enough code) to figure out what the best approach is.

Reply:
should not all set the content type? do we have endpoints that are not json?

Reply:
content-type is set to designate the type of the _request_ body; in the case of GET or DELETE operations, sending that header is unnecessary (since there's no body)

I've taken the use of `Accept` headers from your PR and added it here.